person_capture/video_io.py

```
@@ -1499,7 +1499,7 @@ def _q(s: str) -> str:
         # Tunables (env overrides). Defaults align with MPC VR feel.
         # Strict LP mode = no CPU/zscale fallback. We'll try one minimal LP chain retry, then error.
         self._lp_minimal = False
-        self._sdr_nits = float(os.getenv("PC_SDR_NITS", "125.0"))     # 80–200 typical
+        self._sdr_nits = float(os.getenv("PC_SDR_NITS", "100.0"))     # 80–200 typical
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)